targets/XPlatCppSdk/source/code/source/playfab/OneDSAndroidHttpPlugin.cpp

```diff
@@ -0,0 +1,109 @@
+#include <stdafx.h>
+
+#ifndef DISABLE_ONEDS_API
+
+#include <playfab/OneDSAndroidHttpPlugin.h>
+
+namespace PlayFab
+{
+    constexpr auto ONEDS_SERVICE_URL = "https://self.events.data.microsoft.com/OneCollector/1.0/";
```

Comment:
please use fully typed constants

Reply:
Need to change all platform to be consistency. Added a backlog. [https://microsoft.visualstudio.com/Xbox/_workitems/edit/21253089](https://microsoft.visualstudio.com/Xbox/_workitems/edit/21253089)

---
In reply to: [273691547](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r273691547) [](ancestors = 273691547)